main.py

```diff
@@ -270,6 +271,11 @@ async def execute_opportunities_webhook(webhook_url, job_message, internship_mes
 
 
 async def main():
+    # If you have just cloned this repository, please call the following function below.
+    # Once you have executed the create() function for the first time, please comment it out for future runs.
+
+    # create()
```

Comment:
Instead of making the user uncomment this line of code, shall we instead use an environment variable or CLI flag to conditionally run the line of code?

Reply:
I like this idea, I can start this tomorrow along with the others!

Reply:
Resolved [here](https://github.com/acmcsufoss/acmOpportunities/pull/19/commits/0a25691c94fa80f551275e70dec5c612b0014058)